src/sass/elements/buttons.scss

```diff
@@ -3,11 +3,11 @@ Provides classes for buttons
  */
 
 .mw-button {
-  @include mw-button($MW_COLOR_BRAND_1, $MW_COLOR_NEUTRAL_6);
+  @include mw-button(#0EBBC9, $MW_COLOR_GRAY_6);
```

Comment:
Why is the hexadecimal color used here instead of a color variable?

Reply:
Good question. Is `#0EBBC9` still something we want to have as a variable?

I think it was used as `MW_COLOR_BRAND_1`. Should we add it as a constant or as a variable here?